### PR TITLE
frontend/fix #3353 ride completed screen UI fixes

### LINKED ISSUE #3353: Ride Completed Screen Minor UI Bugs
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior.

**Expected behavior**

**Screenshots**

**App (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by S-PRAVINKUMAR**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

- Some Padding, margins and alignments are fixed

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

**Review by Vignesh-772 — DISMISSED** [review]

**Review by ashishsingh101 — APPROVED** [review]

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs [hunk 1]
```diff
@@ -1839,6 +1839,8 @@ eval (MenuButtonActionController (MenuButtonController.OnClick config)) state =
       _ <- animateCamera config.lat config.lng 25 "NO_ZOOM"
       pure NoAction
     ]
+eval (ChooseYourRideAction (ChooseYourRideController.ChooseVehicleAC ChooseVehicleController.NoAction)) state =
```
Comment (MercyQueen) [inline Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:1842]: @S-PRAVINKUMAR why is this change required?

File: Frontend/ui-common/src/Components/RideCompletedCard/View.purs [hunk 2]
```diff
@@ -45,22 +45,34 @@ topCardView config push =
...
-          , imageWithFallback $ "ny_ic_headphone," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_headphone.png"
-          , onClick push $ const Support
-          ]
+      ][  
+          linearLayout[
+            width MATCH_PARENT
+          , height WRAP_CONTENT
+          , gravity CENTER 
+          , margin $ MarginTop 5
+          ][if config.topCard.topPill.visible then topPillView config push else dummyTextView]
```
Comment (MercyQueen) [inline Frontend/ui-common/src/Components/RideCompletedCard/View.purs:58]: @S-PRAVINKUMAR give proper indentation here.

Comment (MercyQueen) [reply Frontend/ui-common/src/Components/RideCompletedCard/View.purs:58]: and is it tested in iOS?

File: Frontend/ui-common/src/Components/RideCompletedCard/View.purs [hunk 3]
```diff
@@ -45,22 +45,34 @@ topCardView config push =
...
+          , gravity RIGHT
+          ][
+          imageView
+            [ height $ V 40
+            , width $ V 40
+            , accessibility config.accessibility
+            , accessibilityHint "Contact Support : Button"
+            , imageWithFallback $ "ny_ic_headphone," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_headphone.png"
+            , onClick push $ const Support
+            , gravity RIGHT
```
Comment (MercyQueen) [inline Frontend/ui-common/src/Components/RideCompletedCard/View.purs:71]: @S-PRAVINKUMAR  remove gravity if not required.

File: Frontend/ui-common/src/Components/RideCompletedCard/View.purs [hunk 4]
```diff
@@ -45,22 +45,34 @@ topCardView config push =
...
-          , accessibilityHint "Contact Support : Button"
-          , imageWithFallback $ "ny_ic_headphone," <> (getCommonAssetStoreLink FunctionCall) <> "ny_ic_headphone.png"
-          , onClick push $ const Support
-          ]
+      ][  
+          linearLayout[
+            width MATCH_PARENT
+          , height WRAP_CONTENT
+          , gravity CENTER 
+          , margin $ MarginTop 5
```
Comment (MercyQueen) [inline Frontend/ui-common/src/Components/RideCompletedCard/View.purs:57]: is this margin required for dummy textView also?

**Review by MercyQueen — CHANGES_REQUESTED** [review]

File: Frontend/ui-common/src/Components/RideCompletedCard/View.purs (hunk 3 above)
Comment (Vignesh-772) [reply Frontend/ui-common/src/Components/RideCompletedCard/View.purs:71]: gravity will not work in imageView

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs (hunk 1 above)
Comment (ashishsingh101) [reply Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs:1842]: @MercyQueen I asked to add this change this is basically to just modify the state when `NoAction` is called after screen render so that choose your ride component will render with the calculated height of card, otherwise its not updating even if we do continue state and the cards/variants are not visible due to zero height

File: Frontend/ui-common/src/Components/RideCompletedCard/View.purs (hunk 2 above)
Comment (S-PRAVINKUMAR) [reply Frontend/ui-common/src/Components/RideCompletedCard/View.purs:58]: @MercyQueen  Yes

File: Frontend/ui-common/src/Components/RideCompletedCard/View.purs (hunk 4 above)
Comment (S-PRAVINKUMAR) [reply Frontend/ui-common/src/Components/RideCompletedCard/View.purs:57]: Yes

**Review by MercyQueen — APPROVED** [review]

**Review by Vignesh-772 — APPROVED** [review]

**Review by ashishsingh101 — APPROVED** [review]
